elephant_vending_machine/__init__.py

```diff
@@ -7,6 +7,11 @@
 from flask import Flask
 
 APP = Flask(__name__)
+APP.config.update(
+    REMOTE_HOSTS=['192.168.1.11', '192.168.1.12', '192.168.1.13'],
```

Comment:
Out of curiosity, why these specific addresses?

Reply:
I just picked some that ended in 1, 2, and 3 since we have 3 screens and usually 192.168.1.1 is the router's IP. When I go to configure the network equipment, I'll set the Pis to have these static addresses.

Reply:
Makes sense